integration/dhclient_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 the u-root Authors. All rights reserved
+// Copyright 2019 the Webboot Authors. All rights reserved
```

Comment:
I forget the ruling on this Ron, should this be changed?

Reply:
I have no clue but it might as well be lower case

Reply:
I will lower case the Webboot

Reply:
actually, after our discussion, I changed the licensing rights back to u-root as this sounds the most applicable.